packages/devtools-services/README.md

```diff
@@ -0,0 +1,13 @@
+## Services
+
+[![Npm version](https://img.shields.io/npm/v/devtools-services.svg)](https://npmjs.org/package/devtools-services)
+
+Shims [Services.jsm](https://searchfox.org/mozilla-central/source/toolkit/modules/Services.jsm)
+
+* `prefs`
+* `Services`
```

Comment:
I think we only export the method of `Services` and not `{Services, prefs}`

Reply:
that makes sense